examples/virtual_staining/vcp_tutorials/quick_start.py

```diff
@@ -0,0 +1,258 @@
+# %% [markdown]
+"""
+# Quick Start: 2D Virtual Staining of A549 Cells
+
+**Estimated time to complete:** 15 minutes
+"""
+
+# %% [markdown]
+"""
+# Learning Goals
+
+* Download the VSCyto2D model.
+* Predict nuclei and plasma membrane from quantitative phase.
+"""
+
+# %% [markdown]
+"""
+# Prerequisites
+Python>=3.10
+
+"""
+
+# %% [markdown]
+"""
+# Introduction
+
+## Model
+
+The VSCyto2D model is a 2D U-Net that predicts and cell nuclei and plasma membrane
+from quantitative label-free images such as quantitative phase, Zernike phase, and brightfield.
+It has been trained on A549, HEK293T, and BJ-5ta cells.
+
+## Example Dataset
+
+The example dataset contains quantitative phase and paired nuclei and plasma membrane fluorescence images of A549 cells.
+It is stored in OME-Zarr format and can be downloaded from
+[here](https://public.czbiohub.org/comp.micro/viscy/VS_datasets/VSCyto2D/test/a549_hoechst_cellmask_test.zarr).
+It has pre-computed statistics for normalization, generated using the `viscy preprocess` CLI.
+
+Refer to our [preprint](https://doi.org/10.1101/2024.05.31.596901) for more details
+about how the dataset and model were generated.
+"""
+
+# %% [markdown]
+"""
+# Setup
+
+The commands below will install the required packages and download the example dataset and model checkpoint.
+It may take a few minutes to download all the files.
+Assuming a Unix-like shell.
+"""
+
+# %%
+# Install VisCy with the optional dependencies for this example
+# See the [repository](https://github.com/mehta-lab/VisCy) for more details
+# !pip install "viscy[metrics,visual]==0.3.0rc2"
```

Comment:
Using a release candidate for now. @mattersoflight I think we can aim for a 'stable' release after testing and merging these tutorials, so the version that is deployed in production can point to it.
cc @edyoshikun